app/plugins/eu.esdihumboldt.hale.app.transform.test/src/eu/esdihumboldt/hale/app/transform/test/ExecuteTest.groovy

```diff
@@ -36,171 +36,339 @@ class ExecuteTest extends GroovyTestCase {
 	private static final String HYDRO_PROJECT = "platform:/plugin/$PLUGIN_NAME/projects/hydro/hydro-basic.halez"
 	private static final String HYDRO_DATA = "platform:/plugin/$PLUGIN_NAME/projects/hydro/hydro-source.gml.gz"
 
+	private static final String MULTI_TYPE_PROJECT = "platform:/plugin/$PLUGIN_NAME/projects/multitype/multi-type.halez"
```

Comment:
In version control you should always store hale projects as `.halex`. The `.halez` format is actually a Zip file containing the project and any local resources.